models/database.js

```diff
@@ -0,0 +1,82 @@
+const debug = require('debug')('database')
+var AWS = require('aws-sdk');
+AWS.config.update({
+  region: "eu-ireland", 
+  endpoint: "http://localhost:8000"
```

Comment:
Could this be an env var so the code doesn't have to be changed when deployed to prod?